service.go

```diff
@@ -30,6 +30,9 @@ type Service struct {
 	Tier                       Tier                         `json:"tier,omitempty"`
 	Timestamps                 Timestamps                   `json:"timestamps"`
 	Tools                      *ToolConnection              `json:"tools,omitempty"`
+
+	Dependencies *ServiceDependenciesConnection `graphql:"-"`
+	Dependents   *ServiceDependentsConnection   `graphql:"-"`
```

Comment:
The only part i'm not happy about is this part - plugging this into the Hydrate function blows up the scope.   Not sure we want to take the on - but if we don't it means this addition is a bit different then our other sub resources like tools and repos.

Reply:
Yeah I agree, it’s blown up the scope by in the past so I think it makes sense to tackle it in its own ticket if we have the time. Should be ok to add later if needed right?

Reply:
Ya the only downside is that if customers go to use the field it will be nil untill they call `service.GetDependents(client, nil)` and `service.GetDependencies(client, nil)`

Which is subpar.  But if we just auto call it for the customer then we'll be querying a ton of data all the time - even for a simple mutation such as "update the API docs path".